### DISCUSSION TITLE
Question: Is there a way to get 2D image of a 3D object at some angle

### DISCUSSION THREAD

**Discussion by jakubMitura14** [JuliaImageRecon/ImagePhantoms.jl#89] [Q&A] [answered]
Hello I have 3d object composed of cylinders and I reconstruct it into 3d voxel array using filtered back projection.
Is there a way to get 8 2D images that are the projection of the 3D object on flat surface every 45 degrees? 
- so I want to have eight 2 dimensional images instead of one 3 dimensional

**Comment by JeffFessler** [comment]
After you apply FBP you no longer have cylinders; you have voxels.
Then you need a discrete forward projector designed for voxels.
See https://github.com/JuliaImageRecon/Sinograms.jl/issues/34 and the package(s) discussed there.

A related visualization is here that illustrates what I think you want:
https://juliaimagerecon.github.io/SPECTrecon.jl/stable/generated/examples/1-overview
But ImagePhantoms is not really the package for voxelized forward projection
(though it could be hacked with a huge grid of cubes).

**Reply by jakubMitura14** [reply]
Thanks for reference as you said after filtered back projection there is no good way, but before ? For simplicity let's say that have a phantom that is a Cylinder and want to have 2D images of rectangles from diffrent angles, so like classical RTG of a 3D object?

**Reply by JeffFessler** [reply, marked as answer]
Oh that's easy.  That's exactly what the Sinograms package and its `radon` function is designed to do.
There is an example of that with ellipsoids here where the 2D images look like semi-ellipsoids:
https://juliaimagerecon.github.io/Sinograms.jl/stable/generated/examples/07-fdk
